lib/helpers/UiBaseLocators.ts

```diff
@@ -122,11 +122,4 @@ export class UiBaseLocators {
   async isTreeItemVisible(name: string) {
     await expect(this.page.locator('umb-tree-item').locator('[label="' + name + '"] ')).toBeVisible();
   }
-
-  async goToSection(sectionName: string) {
-    for (let section in ConstantHelper.sections) {
-      await expect(this.page.getByRole('tab', {name: section})).toBeVisible();
-    }
-    await this.page.getByRole('tab', {name: sectionName}).click();
```

Comment:
Why are we moving this from the UiBase to the UiHelpers?

Reply:
In the tests, we still need to directly call the goToSection() function after logging in (for example, in tests related to Dictionary, Members, Users, and Packages). Therefore, I believe it should be placed in this file, similar to goToSettingsTreeItem(). If we were to place it in UiBase, you would need to wrap it in each test, resulting in redundant code.

Reply:
Makes good sense!